WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/CategorySectionTableViewCell.xib

```diff
@@ -52,14 +52,15 @@
                     <constraint firstItem="mQ0-DH-hTW" firstAttribute="top" secondItem="3pe-2p-9as" secondAttribute="bottom" constant="16" id="Lj1-O3-zk4"/>
                     <constraint firstItem="3pe-2p-9as" firstAttribute="top" secondItem="H2p-sc-9uM" secondAttribute="top" constant="20" id="Mrq-dy-dGi"/>
                     <constraint firstAttribute="trailing" secondItem="mQ0-DH-hTW" secondAttribute="trailing" id="XIg-vD-05h"/>
-                    <constraint firstAttribute="bottom" relation="greaterThanOrEqual" secondItem="mQ0-DH-hTW" secondAttribute="bottom" constant="30" id="aLq-Bg-Bfl"/>
+                    <constraint firstAttribute="bottom" secondItem="mQ0-DH-hTW" secondAttribute="bottom" constant="30" id="aLq-Bg-Bfl"/>
```

Comment:
This may not have been necessary to change, but it seems to be a little more deterministic to always be 30.